java/src/jmri/jmrix/loconet/LnPacketizerStrict.java

```diff
@@ -181,7 +181,7 @@ public void run() {
                 } catch (LocoNetMessageException e) {
                     // just let it ride for now
                     log.warn("run: unexpected LocoNetMessageException", e); // NOI18N
-                } catch (java.io.EOFException e) {
+                } catch (java.io.EOFException | com.fazecast.jSerialComm.SerialPortTimeoutException e) {
```

Comment:
Thanks for doing this.

Do you have a stack trace when this exception happens?

The reason I ask is that we want to keep the serial port implementation to the class jmri.jmrix.AbstractPortController.java. So it would be better to have this code catch a generic exception. But to fix that, we need to know where this exception is thrown.

Reply:
This just makes the same change that was made in the LnPacketizer. (line 311) and happens on serail timeout. So the message reported was "End Of File", followed by null ostream as it fell thru to the java.io.IOException.

Reply:
According to [the Javadoc](https://fazecast.github.io/jSerialComm/javadoc/com/fazecast/jSerialComm/SerialPortTimeoutException.html), `SerialComm.SerialPortTimeoutException` inherits from `java.io.InterruptedIOException`.  Perhaps catch that?

Reply:
I am doing what was done in LnPacketizer, is that wrong too?

Reply:
The issue is that it's desired to keep all references to `com.fazecast.jSerialComm` to the class `jmri.jmrix.AbstractPortController`. It makes it much easier if we need to move to another serial library. See for example PR #13048. So yes, it would be good to fix LnPacketizer too.

@bobjacobsen 
Can you understand why ArchitectureTest doesn't catch these? I'm surprised that ArchitectureTest accept this.

Reply:
I don't understand. I copied code that you approved in 13035 last week!

Reply:
Done